File: src/main/java/leets/leenk/domain/leenk/application/exception/CannotLeaveAsHostException.java
package leets.leenk.domain.leenk.application.exception;

import leets.leenk.global.common.exception.BaseException;

public class CannotLeaveAsHostException extends BaseException {
    public CannotLeaveAsHostException() {
        super(ErrorCode.LEENK_CANNOT_LEAVE_AS_HOST);
    }
}

File: src/main/kotlin/leets/leenk/domain/leenk/application/exception/CannotLeaveAsHostException.kt
package leets.leenk.domain.leenk.application.exception;

import leets.leenk.global.common.exception.BaseException;

public class CannotLeaveAsHostException extends BaseException {
    public CannotLeaveAsHostException() {
        super(LeenkErrorCode.LEENK_CANNOT_LEAVE_AS_HOST);
    }
}
